vulture_allowlist.py

```diff
@@ -102,6 +102,9 @@
 app.url_map.strict_slashes
 api.representations
 hub.exception_stream
+app.login_via_request
+app.should_set_cookie
```

Comment:
Shouldn't this be `CustomSessionInterface.should_set_cookie`? What does the `app` have to do with it?

Reply:
The vulture was complaining about the app.